### ISSUE TITLE
Error on install- can't enable plugin: rclone.sock no such file or directory

### ISSUE THREAD

**Issue by Rojikku** [sapk/docker-volume-rclone#38]
- Plugin version (or commit ref) :
- Docker version : Docker version 19.03.12-ce, build 48a66213fe
- Plugin type : legacy/managed
`Legacy plugins do not work in swarm mode. However, plugins written using the v2 plugin system do work in swarm mode, as long as they are installed on each swarm worker node.`
- Operating system: ubuntu 20.04 on RPI4 4GB

## Description

I am running in swarm mode. If the plugin type is actually legacy, as it defaults to in this template, then that explains why this doesn't work.
Though if I didn't want to run in swarm mode, I could literally just give it cap_add SYS_ADMIN and be done with it- at least that works in kubernetes, so I assume I could get it to work in swarm.

## Logs

```
 $ docker plugin install sapk/plugin-rclone
Plugin "sapk/plugin-rclone" is requesting the following privileges:
 - network: [host]
 - device: [/dev/fuse]
 - capabilities: [CAP_SYS_ADMIN]
Do you grant the above permissions? [y/N] y
latest: Pulling from sapk/plugin-rclone
e7d99237d18e: Download complete 
Digest: sha256:f4a9025fbb59f70f9d3fb58ad092371accfd08370a5de60c813c73cc773ae803
Status: Downloaded newer image for sapk/plugin-rclone:latest
Error response from daemon: dial unix /run/docker/plugins/e56de56989076030fdd8c57952983ce859c2bd1d78abf753daa7716e4cf08468/rclone.sock: connect: no such file or directory
```
The bottom line repeats if I run the `docker plugin enable sapk/plugin-rclone` command

## Tests

`make: *** No rule to make target 'test-integration'.  Stop.`
Yeah uh..I see a go file that should be it, but honestly, don't know how to use it off the top of my head. This is probably because legacy instead of v2, so I'll leave it there.

**Comment by sapk** [comment]
Hi, sorry for the late response. 

I think the problem come from that you are using it on an RPI (arm). 
The image of the plugin is not (yet) multi-arch.

**Comment by adlion-gofore** [comment]
I am in arm and have same issue

**Comment by Yundin** [comment]
Got the same issue
Removing content of `/var/lib/docker-plugins/rclone/cache` and enabling back the plugin works
